### ISSUE TITLE
Analog Sticks are not currently supported

### ISSUE THREAD

**Issue by zlittell** [zlittell/MSF-FightStick#4]
Add support for filling in the bytes for analog sticks in the report.  Use some of @Jerware fancy code to split the words into bytes to make the process more automated for others.

**Comment by zlittell** [comment]
Solved by commit 7808eb2 in the new features branch

**Comment by Jerware** [comment]
Not necessary for me, but maybe also include support for analog triggers while you're at it? Looks like they're only 8 bit values.

From free60.org:

```
Offset   Length (bits)   Description     Windows driver
0x04.0   8   Left trigger    Z-axis down
0x05.0   8   Right trigger   Z-axis up
```
